.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules
 
-# Compiled code
+# (previously) Compiled code
 /lib/
```

Comment:
```suggestion
```
why not just remove the entry entirely?

Reply:
Eh, just in case anyone has contributed previously and has files hanging around, to prevent them from accidentally being checked in.  Figured it doesn't hurt anything.